db/migrate/20230316050736_create_tags.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTags < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tags do |t|
+      t.text :tag_name
```

Comment:
Usually, you don't want to include the name of the table in the name of the column. What do you think about changing this to `name`?

Reply:
yes makes sense, fixed